apps/medusa-be/.env.template

```
@@ -22,10 +22,13 @@ MINIO_ACCESS_KEY=supersecret
 MINIO_REGION=us-east-1
 
 # CORS and admin host
+# Public storefront base URL (used to build links in transactional emails)
+STOREFRONT_URL=http://localhost:3000
 STORE_CORS=http://localhost:8000,http://localhost:9000,http://localhost
 ADMIN_CORS=http://localhost:5173,http://localhost:9000
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The ADMIN_CORS key should go before the STOREFRONT_URL key

(UnorderedKey)